js/main.js

```diff
@@ -26,4 +26,102 @@ $form.addEventListener('submit', function (event) {
 
   $form.reset();
   $newEntryImg.setAttribute('src', 'images/placeholder-image-square.jpg');
+
+  $noEntries.remove();
+  i = 0;
+  $entryGallery.prepend(addEntry());
+
+  for (i = 0; i < allViews.length; i++) {
+    if (allViews[i].getAttribute('data-view') === 'entries') {
+      allViews[i].className = 'view';
+    } else {
+      allViews[i].className = 'view hidden';
+    }
+  }
```

Comment:
You have this sort of behavior repeated a few times in your code. This would be a good opportunity to make a function whose sole job is to change the view. You can give the function a parameter, which will be a string holding the name of that view